js/electron/transcoding.js

```diff
@@ -307,13 +438,38 @@ function transcodingProcessor(electronIpcMain) {
       try {
         buffer = await promise;
       } catch (err) {
-        if (err === 'TRANSCODE_ABORT') {
-          const err = Error('TRANSCODE_ABORT');
-          e.sender.send('transcode-video-error', err);
-          return;
+        switch (err) {
+          case 'TRANSCODE_ABORT':
+            const errTranscode = Error(err);
+
+            console.log('Transcode aborted');
+            e.sender.send('transcode-video-error', errTranscode);
+            break;
+          case 'ERROR_FILE_ACCESS':
+            const errAccess = Error(err);
+
+            console.log('Transcoding error: file access error');
+            e.sender.send('transcode-video-error', errAccess);
+            return;
+          case 'FFBIN_NOT_DOWNLOADED':
+            const errFfbin = Error(err);
+
+            console.log('Transcoding error: binaries still not downloaded');
+            e.sender.send('transcode-video-error', errFfbin);
+            return;
+          case 'VERTICAL_VIDEO_NOT_SUPPORTED':
+            const verticalNotSupported = Error(err);
+
+            console.log('Transcoding error: vertical video not supported');
+            e.sender.send('transcode-video-error', verticalNotSupported);
+            return;
+          default:
+            const errUnhandled = Error('UNHANDLED_ERROR');
+
+            console.error(err);
+            e.sender.send('transcode-video-error', errUnhandled);
+            return;
         }
```

Comment:
### More errors handled here
- **Transcode abort** - happens on user canceled transcoding.
- **ErrorFileAccess** - happens on error with accessing video.
- **FFBinNotDownloaded** - happens when FF Binaries download error occurred.
- **VerticalVideo** - happens when video is vertical
- **UnhandledError** - happens on some unhandled error. Logging it with `console.error` for debugging.